opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -601,24 +606,26 @@ export const taskHandler = async () => {
       task_processed_number: processedNumber,
       completed: processingElements.length < MAX_TASK_ELEMENTS,
     };
+    logApp.info('[OPENCTI-MODULE][TASK-MANAGER] Elements processing done, store task status.', { patch });
     await updateTask(context, task.id, patch);
+    const totalTime = new Date().getMilliseconds() - startingTime;
+    logApp.info(`[OPENCTI-MODULE][TASK-MANAGER] Task manager done in ${totalTime} ms`);
   } catch (e) {
     if (e.name === TYPE_LOCK_ERROR) {
-      logApp.debug('[OPENCTI-MODULE] Task manager already in progress by another API');
+      logApp.info('[OPENCTI-MODULE][TASK-MANAGER] Task manager already in progress by another API');
```

Comment:
I'm not really sure we would need this log in info, since we'll see it a lot for all other ingest nodes that will try to run the task manager but won't have the lock (once every 10s for each node, except the one which has the lock)

Reply:
```suggestion
      logApp.debug('[OPENCTI-MODULE] Task manager already in progress by another API');
```